patch/driver-arista-i2c-designware-shutdown.patch

```diff
@@ -0,0 +1,49 @@
+From: Justin Oliver <justinoliver@arista.com>
+Date: 2022-12-12 17:42:20 +0000
+Subject: Designware I2C driver fails to probe device after kexec
+
+The Designware I2C driver sometimes fails to probe the I2C controller after
```

Comment:
Please add the error log message from the merge/pull request description.

Reply:
Done